roles/logging/templates/etc/rsyslog.d/60-remote-syslog.conf

```diff
@@ -0,0 +1,17 @@
+$WorkDirectory {{ logging.syslog_forwarding.work_directory }}
+
+$ActionResumeRetryCount {{ logging.syslog_forwarding.action_resume.retry_count }}
```

Comment:
the template is currently fragile in that if `logging.syslog_forwarding.action_resume.retry_count` were to be undefined we would still be rendering the `$ActionResumeRetryCount` config line.
Can you wrap each config line in a {% if logging.syslog_forwarding.action_resume.retry_count is defined` %} ?